RoosterPlanner.UI/src/app/components/add-task/add-task.component.ts

```diff
@@ -55,11 +58,29 @@ export class AddTaskComponent implements OnInit {
     })
   }
 
-  saveTask(value: Task) {
+  async saveTask(value: Task) {
     this.updatedTask = value;
     if (this.checkoutForm.status === 'INVALID') {
       this.toastr.error("Niet alle velden zijn correct ingevuld")
     } else {
+
+      if (this.files != null) {
```

Comment:
kan gecombineerd worden in een enkele check: if(this.files) gezien null, undefinied en lege array allemaal falsey zijn.